### PR TITLE
Refactoring/tao 5104/remove globals from generis

### PR DESCRIPTION
**Pull request by ionutpad**

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
   'label'       => 'Delivery Management',
   'description' => 'Manages deliveries using the ontology',
   'license'     => 'GPL-2.0',
-  'version'     => '3.22.0',
+  'version'     => '3.23.0',
 	'author'      => 'Open Assessment Technologies SA',
 	'requires'    => array(
         'generis'     => '>=5.11.0',
```
Comment (zagovorichev) [inline manifest.php]: depends on tao and generis (new version)

Comment (ionutpad) [reply manifest.php]: In this case a downgrade it's necessary?

Comment (zagovorichev) [reply manifest.php]: of course not, I mixed generis with tao, sorry

**Comment by zagovorichev** [comment]
@ionutpad could you rollback your last commit, please

**Pull request merged**